lecture_note.tex

```diff
@@ -881,7 +881,7 @@ \subsection{Early Stopping}
 and the validation cost as
 \begin{align}
     \label{eq:val_c}
-    C_{\val}(\TT) = \frac{1}{|D_{\val}|} \sum_{(x,y) \in D_{\val}} D(\hat{y}, y).
+    C_{\val}(\TT) = \frac{1}{|D_{\val}|} \sum_{(x,y) \in D_{\val}} D_{\var}(\hat{y}, y).

```

Comment:
In fact, this should stay same, but the second ``D_{\train}`` in the line 879 should be changed to ``D``